spec/system/placements/schools/placements/school_user_edits_a_placement_with_no_mentors_spec.rb

```diff
@@ -0,0 +1,202 @@
+require "rails_helper"
+
+RSpec.describe "School user edits a placement with no mentors", service: :placements, type: :system do
+  scenario do
+    given_that_placements_exist
+    and_i_am_signed_in
+
+    when_i_am_on_the_placements_index_page
+    then_i_see_my_placement
+
+    when_i_click_on_my_placement
+    then_i_see_the_placement_details_page
+    and_i_see_the_add_a_mentor_link
+
+    when_i_click_on_add_a_mentor
+    then_i_see_the_mentors_index_page
+
+    when_i_click_on_add_mentor
+    then_i_see_the_find_teacher_page
+
+    when_i_fill_in_the_details
+    and_i_click_on_continue
+    then_i_see_the_confirm_details_page
+
+    when_i_click_on_confirm_and_add_mentor
+    then_i_see_the_mentors_index_page
+    and_i_see_a_success_message
+
+    when_i_click_on_the_placements_navigation_item
+    then_i_see_the_placements_index_page
+
+    when_i_click_on_my_placement
+    then_i_see_the_placement_details_page_with_assign_a_mentor_text
+  end
+
+  private
+
+  def given_that_placements_exist
+    @springfield_elementary_school = build(
+      :placements_school,
+      name: "Springfield Elementary",
+      address1: "Westgate Street",
+      address2: "Hackney",
+      postcode: "E8 3RL",
+      group: "Local authority maintained schools",
+      phase: "Primary",
+      gender: "Mixed",
+      minimum_age: 3,
+      maximum_age: 11,
+      religious_character: "Does not apply",
+      admissions_policy: "Not applicable",
+      urban_or_rural: "(England/Wales) Urban major conurbation",
+      percentage_free_school_meals: 15,
+      rating: "Outstanding",
+    )
+
+    @primary_english_subject = build(:subject, name: "Primary with english", subject_area: :primary)
+
+    @autumn_term = build(:placements_term, name: "Autumn term")
+    @spring_term = create(:placements_term, name: "Spring term")
+    @summer_term = create(:placements_term, name: "Summer term")
```

Comment:
Do we all need all of these?

Reply:
Yes, we should set up the data as if it were in production as far as possible, these terms are generated via the seeds file so it's a fair representation!

Reply:
Even if given this scenario the other terms will never been seen or used?